src/components/formik-select/FormikSelect.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * @class FormikSelect
+ */
+
+import * as React from "react";
+import styled from "styled-components";
+import { loadStyleVariables } from "../../scripts/loadStyleVariables";
+
+interface Option {
+  option: string,
+}
+
+type Props = {
+  field: {
+    name: string;
+  }
+  form: {
+    ['errors']: string;
+    ['touched']: boolean;
+  };
+
+  ['serverErrors']: string;
+  userInput?: string,
+  options: Option[],
+  label?: string,
+  optionSet?: any,
```

Comment:
optionSet wordt nergens echt gebruikt. Het is ook niet clean dat je een soort van 'configuratie'-prop mee zou moeten geven.